uvm/servlets/admin/app/view/main/MainController.js

```diff
@@ -120,7 +120,7 @@ Ext.define('Ung.view.main.MainController', {
     },
 
     suggestHandler: function (btn) {
-        var suggestUrl = Rpc.directData('rpc.helpUrl') + '?fragment=feedback&' + Util.getAbout();
+        var suggestUrl = Rpc.directData('rpc.feedbackUrl');
         window.open(suggestUrl);
```

Comment:
Add the same parameters to the new URL.

Reply:
With parameters it is navigating to page shown below, with message : **Nothing found for this page. Try a search instead?**

I think those parameters were required for wiki link to navigate to correct page as  same wiki link was used for help and feedback.

![Screenshot from 2024-01-31 11-45-04](https://github.com/untangle/ngfw_src/assets/154422821/9c813f46-4554-4752-97c7-454e4759091a)

Reply:
Got it.  I see the problem.  Ok, this PR is approved.